components/developer/gdb/gdb.p5m

```diff
@@ -34,9 +35,17 @@ set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/bin/gdb
+
+file path=usr/bin/gcore
```

Comment:
Unfortunately, this will very likely clash with `developer/linker`:
```
# pkg search gcore
INDEX      ACTION   VALUE               PACKAGE
basename   file     usr/bin/i86/gcore   pkg:/developer/linker@0.5.11-2022.0.0.21323
basename   file     usr/bin/amd64/gcore pkg:/developer/linker@0.5.11-2022.0.0.21323
basename   file     usr/bin/i86/gcore   pkg:/developer/linker@0.5.11-2022.0.0.21323
basename   file     usr/bin/amd64/gcore pkg:/developer/linker@0.5.11-2022.0.0.21323
basename   hardlink usr/bin/gcore       pkg:/developer/linker@0.5.11-2022.0.0.21323
#
```

Reply:
Well then we can remove it. Which is why it wasn't added before